sycl/test/hier_par/hier_par_basic.cpp

```diff
@@ -12,6 +12,9 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 
+// The tets is failing on CPU RT 2020.10.4.0.15 an above
+// XFAIL: cpu
```

Comment:
I assume that tests are disabled due to bugs in the drivers.
Why should we switch to buggy drivers instead of using stable versions?

Reply:
We should be aligned with the drivers which are used for the release. 
+@pvchupin to comment

Reply:
The same question applies to "the release".
What is the value? New features? More critical bugs are fixed?